www/src/data/sidebars/doc-links.yaml

```
@@ -296,6 +294,8 @@
       link: /docs/prpl-pattern/
     - title: Querying data with GraphQL
       link: /docs/querying-with-graphql/
+    - title: Client data fetching
```

```suggestion
    - title: Fetching client data
```
IMHO, should this line also be consistent ?